Mathlib/Analysis/Asymptotics/TVS.lean

```diff
@@ -100,22 +148,60 @@ theorem _root_.Filter.HasBasis.isLittleOTVS_iff {ιE ιF : Sort*} {pE : ιE →
   · refine fun s t hsub h ε hε ↦ (h ε hε).mono fun x hx ↦ hx.trans ?_
     gcongr
 
+theorem isLittleOTVS_iff_smallSets :
+    f =o[𝕜; l] g ↔ ∀ U ∈ 𝓝 0, ∀ᶠ V in (𝓝 0).smallSets, ∀ ε ≠ (0 : ℝ≥0),
```

Comment:
Just a remark that I'd rather that what is called `ε` here was instead `C` or really anything other than `ε` (or `δ`) as this constant will generally be large.

I note that you're being consistent with the definition `IsLittleOTVS` so I guess I'm asking how you would feel about following up with a renaming PR to rename the constant there?